src/vellum/workflows/workflows/base.py

```diff
@@ -255,9 +255,41 @@ def stream(
             external_inputs=external_inputs,
             cancel_signal=cancel_signal,
         ).stream():
-            if WorkflowEventType(event.name.split(".")[0].upper()) in event_types:
+            if should_yield(event):
                 yield event
 
+    def DEFAULT_EVENT_FILTER(self, event: WorkflowEvent) -> bool:
```

Comment:
nit: We allow for a user pass in a callable of their own. This callable of theirs likely won't have access to `self` right? And will be more akin to a static method? If so, then it'd be nice for our default callable to be more analogous to that which a user might pass. i.e. not live on this class and just be static helper methods.

In this world especially (but also I think in general) I'd prefer to name these:
`def workflow_event_filter` (the fact that it's declared above in `or` implies that it's the default imo, so would rather the name be more indicative of what it actually does)

`def root_workflow_event_filter`

And then also incldue docstrings for each.